README.md

```diff
@@ -5,7 +5,7 @@ npm install
 ```
 
 ## Configure
-create a local configuration file, and fill in credentials:
+By default the configuration is set to read from environment variables `OKAPI_BASE_URL`, `OKAPI_USERNAME`, and `OKAPI_PASSWORD`, but you can overwrite this config by creating `config/local.json` and setting values you wish to use.
```

Comment:
Shouldn't we replace `config/default.json` if the envvar version is going to be the new default? I'm a bit confused about how this works

Reply:
This is how the config-node works. You have specify envionment vars in a file named `config/custom-environment-variables.json` https://github.com/node-config/node-config/wiki/Environment-Variables#custom-environment-variables

Reply:
Oh, I didn't realize it was a magic name. thx